### DISCUSSION TITLE
GitHub projects subtasks

### DISCUSSION THREAD

**Discussion by matt-gust** [community/community#120147] [Projects and Issues] [answered]
### Select Topic Area

Question

### Feature Area

Projects

### Body

Hello,

I recently started using GitHub projects and really enjoying it so far. One feature I'd like to see is being able to add subtasks below a task within a project.

For instance, in the picture below from a microsoft planner project the "main task" is "meet either 4/17 or 4/18" and the "subtasks" would be everything underneath it. Is there a similar feature in GitHub projects? Or, is there a template that allows you to do this? 

Thanks,

Matt

![image](https://github.com/community/community/assets/138897336/de968821-2e57-4d5c-8ff8-3e79e3c392be)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by beingminimal** [comment, marked as answer]
@matt-gust Following two things are under development.
https://github.com/github/roadmap/issues/837
https://github.com/github/roadmap/issues/927

**Reply by matt-gust** [reply]
Great, thanks! Looking forward to these features.
